engine-primitives/engine-primitives/attributes.go

```diff
@@ -45,6 +46,10 @@ type PayloadAttributes struct {
 	// to the block currently being processed. This field was added for
 	// EIP-4788.
 	ParentBeaconBlockRoot common.Root `json:"parentBeaconBlockRoot"`
+
+	// PrevProposerPubKey carries the public key of previous block proposed
+	// Needed to enshrine distributeFor in EVM
```

Comment:
```suggestion
	// This field was added for BRIP-0004.
```
nit: following the pattern for the ParentBeaconBlockRoot field

Reply:
Done